packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -199,16 +199,22 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
 
   public dismissSuggestions = (ev?: any): void => {
     const selectItemFunction = () => {
+      let addItemOnDismimss: boolean | void = true;
       if (this.props.onDismiss) {
-        this.props.onDismiss(
+        addItemOnDismimss = this.props.onDismiss(
```

Comment:
instead of making `onDismiss` to return a `boolean` (un-intuitive interface to have for `onDismiss`), why can't you just call `ev.preventDefault();` in your `onDismiss` handler?

Reply:
It's definitely weird, but it looks like from #10892 and my own personal experimentation, preventDefault doesn't actually prevent all events from performing their default behavior. I think @joschect suggested modifying onDismiss to return a boolean to work around that.

Reply:
i see. okay